renovate.json

```diff
@@ -7,7 +7,7 @@
     {
       "customType": "regex",
       "fileMatch": [
-        "^build-process.yml$"
+        "^.github\\workflows\\build-process.yml$"
```

Comment:
ℹ️ I wasn't expecting the entire path to be necessary but it totally makes sense that it would be.